### PR TITLE
[Outlook] (append on send) Calling in ItemSend handler not supported

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[995e8a1](https://github.com/OfficeDev/office-js-docs-pr/commits/995e8a1c606adaeaccd29c3c02b6fca40c2e2658)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/outlook/append-on-send.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-append-error/docs/outlook/append-on-send.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/outlook/append-on-send?branch=pr-en-us-2075) |
[docs/outlook/outlook-on-send-addins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-append-error/docs/outlook/outlook-on-send-addins.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/outlook/outlook-on-send-addins?branch=pr-en-us-2075) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C8%5C7%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202008071828003948-2075%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=ZLWV%2Bekl%2FKStmphyTL2UXOqIxOfxU7EqxNzLuSUZBSU%3D&st=2020-08-07T18%3A23%3A46Z&se=2020-09-07T18%3A28%3A46Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by o365devx** [comment]
Docs Build status updates of commit _[937ac5e](https://github.com/OfficeDev/office-js-docs-pr/commits/937ac5e1907e1c8cba127530ff9daf7dba7052af)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/outlook/append-on-send.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-append-error/docs/outlook/append-on-send.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/outlook/append-on-send?branch=pr-en-us-2075) |
[docs/outlook/outlook-on-send-addins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-append-error/docs/outlook/outlook-on-send-addins.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/outlook/outlook-on-send-addins?branch=pr-en-us-2075) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C8%5C7%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202008071830006835-2075%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=AH3DgZ16DRQ0blAsaprYxgllITaBTzr4U12iSdMKIP4%3D&st=2020-08-07T18%3A25%3A32Z&se=2020-09-07T18%3A30%3A32Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/outlook/outlook-on-send-addins.md [hunk 1]
```diff
@@ -63,6 +63,7 @@ The following screenshot shows an information bar that notifies the sender that
 
 The on-send feature currently has the following limitations.
 
+- **Append-on-send** feature (preview) &ndash; If you call `item.body.AppendOnSendAsync` in the on-send handler, an error is returned.
```
Comment (emtencza) [inline docs/outlook/outlook-on-send-addins.md:66]: nit: appendOnSendAsync

**Comment by o365devx** [comment]
Docs Build status updates of commit _[1553f6b](https://github.com/OfficeDev/office-js-docs-pr/commits/1553f6beee9353b160eab16f6901a779776718a0)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/outlook/append-on-send.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-append-error/docs/outlook/append-on-send.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/outlook/append-on-send?branch=pr-en-us-2075) |
[docs/outlook/outlook-on-send-addins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-append-error/docs/outlook/outlook-on-send-addins.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/outlook/outlook-on-send-addins?branch=pr-en-us-2075) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C8%5C7%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202008072026349083-2075%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=Hk1pQL4gL7GN6nT8IIBtODUuuBc8K87hXveRsDAfKa4%3D&st=2020-08-07T20%3A22%3A17Z&se=2020-09-07T20%3A27%3A17Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Pull request merged**